vendor/vtd/conanfile.py

```diff
@@ -43,17 +46,48 @@ def configure(self):
     def build(self):
         src = Path(self.source_folder)
         dst = Path(self.build_folder)
+        vtddir = dst / "VTD.2.2"
+        libdir = vtddir / "lib"
 
         def extract_archive(archive):
             print(f"Extracting: {archive}")
             tools.untargz(src / archive, dst)
 
         extract_archive(self._archive_base)
+        libdir.mkdir()
         if self.options.with_osi:
             extract_archive(self._archive_osi)
         if self.options.with_road_designer:
             extract_archive(self._archive_rod)
 
+        # Get a list of all binaries we need to set the RPATH of:
+        result = subprocess.run(
+            """find -type f -executable -exec sh -c "file -i '{}' | grep -q '; charset=binary'" \; -print""",
+            shell=True,
+            stdout=subprocess.PIPE,
+            check=True,
+        )
+        binary_files = [Path(x) for x in result.stdout.decode().splitlines()]
+        # Set RPATH of all the collected binaries:
+        for file in binary_files:
+            assert file.exists()
+            rpath = os.path.relpath(libdir, (dst / file).parent)
+            try:
+                print(f"Setting RPATH: {file} -> $ORIGIN/{rpath}")
+                result = subprocess.run(
+                    ["patchelf", "--set-rpath", f"$ORIGIN/{rpath}", str(file)],
+                    check=True,
+                )
+            except:
+                print(f"Error: cannot set RPATH of {file}", file=sys.stderr)
+
+        # Bundle libraries that we need at runtime so that this package is portable:
+        with Path(src / "library_filelist.txt").open() as file:
+            libraries = [Path(x.strip()) for x in file.read().splitlines()]
+        for path in libraries:
+            print(f"Bundling system library: {path}")
+            shutil.copy(path, libdir, follow_symlinks=False)
```

Comment:
This does not work on my machine since `library_filelist.txt` contains a bunch of libraries that I don't have or where I have a different version.

Reply:
This is a problem; one way around this is to only specify the symlinks to copy and resolve these into concrete paths. I could do this.

Reply:
I implemented this now, so it should work a little better.